dev/CONTRIBUTING.md

```diff
@@ -100,87 +94,60 @@ Run `builder run check` before committing to ensure lint and tests are passing.
+
+### Consistency
+
+Similar props across Victory components should have the same names and operate similarly. For example, the style prop should almost always have the shape:
+
+```
 style={{
-  parent: {
-    border: "1px solid #ccc",
-    height: 500,
-    margin: 50,
-    width: 500
-  },
   data: {
-    fill: "red",
-    opacity: 0.8
+    ...
   },
   labels: {
-    fontSize: 15,
-    padding: 20,
-    fill: "grey"
+    ...
   }
 }}
 ```
 
-- Support animating and static visualizations
-- Take an `animate` prop as an object that can be passed as props to [victory-animation](https://github.com/FormidableLabs/victory-animation)
-- Support rendering svg and g tags via a `standalone` prop (renders svg when true, g when false).
-- Not require users to write d3 or have d3 domain knowledge
-- Be accessible and usable for junior developers
-- Have an intuitive api
+When writing a new component please reference other Victory components when deciding on a set of props. When writing a component for use with VictoryChart, the API should close to a strict superset of the props described in other components like [`VictoryBar`](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-bar/victory-bar.js#L46)
+
+### Animation 
+
+Victory Components animate via a flexible animation wrapper `VictoryTransition` which handles load animations and entrance and exit transitions before interpolation between sets of props with `VictoryAnimation`. All Victory components that render elements (not HOCs that are only responsible for coodinating other components) should return the component wrapped in `VictoryTransition` as in this [example](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-scatter/victory-scatter.js#L469). Components should also expose their set of transiton parameters via a static method. Most components will use standard transitions: either discrete [as in VictoryScatter](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-scatter/victory-scatter.js#L22) or continuous [as in VictoryLine](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-line/victory-line.js#L21). It also possible to define custom default transitions where appropriate. For example [VictoryBar defines custom transitions](https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-bar/victory-bar.js#L29) which cause the barse to rise from zero. These transitions and other animation properties can all be defined by users via the `animate` prop.
```

Comment:
I wasn't sure what "HOCs" was at first. I Googled it, then looked at the context again and it clicked. I'd just spell it out :)